File: app-miyou/src/main/java/com/miyou/app/application/monitoring/port/ConversationMetricsPort.java
package com.miyou.app.application.monitoring.port;

public interface ConversationMetricsPort {
	void recordConversationIncrement();

	void recordConversationReset();

	void recordQueryLength(int length);

	void recordResponseLength(int length);

	void recordConversationCount(long count);
}

File: application/src/main/kotlin/com/miyou/app/application/monitoring/port/ConversationMetricsPort.kt
package com.miyou.app.application.monitoring.port

interface ConversationMetricsPort {
    fun recordConversationIncrement()

    fun recordConversationReset()

    fun recordQueryLength(length: Int)

    fun recordResponseLength(length: Int)

    fun recordConversationCount(count: Long)
}
